src/main/kotlin/org/rust/lang/core/resolve/NameResolution.kt

```diff
@@ -887,6 +887,7 @@ fun processAssocTypeVariants(trait: RsTraitItem, processor: RsResolveProcessor):
 fun processMacroCallPathResolveVariants(path: RsPath, isCompletion: Boolean, processor: RsResolveProcessor): Boolean {
     val qualifier = path.qualifier
     return if (qualifier == null) {
+        if (path.hasColonColon) return false
```

Comment:
Hmm, IIRC on 2015 edition it's possible to call a `#[macro_export]` macro declared in the crate root using `::foo!()`

Reply:
You are right, added check for 2018 edition